index.rb

```diff
@@ -5,6 +5,10 @@ class MyList
   def initialize(*enumerables)
     @enumerables = enumerables
   end
+
+  def each(&block)
+    @enumerables.each(&block)
+  end
 end
```

Comment:
- [ ] `Each` method should `yield` all members of `@enumerables` in succession
Remember that we use `yield` when we want to call any method with the block code